<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLScalarsDisplayWidget</class>
 <widget class="QWidget" name="qMRMLScalarsDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scalars Display</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="ScalarsVisibilityLabel">
     <property name="text">
      <string>Visible:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="ScalarsVisibilityCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ActiveScalarLabel">
     <property name="text">
      <string>Active Scalar:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="ctkVTKDataSetArrayComboBox" name="ActiveScalarComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ScalarsColorTableLabel">
     <property name="text">
      <string>Color Table:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="qMRMLColorTableComboBox" name="ScalarsColorNodeComboBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ScalarRangeModeLabel">
     <property name="text">
      <string>Scalar Range Mode:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="DisplayedScalarRangeModeComboBox">
     <property name="toolTip">
      <string>Set the scalar range mode flag. Data (auto) updates the Display Scalar Range from the scalar range of the data. Direct color mapping option bypasses lookup table and uses scalar values as RGB color.</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Manual</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Data scalar range (auto)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Color table (LUT)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Data type</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Direct color mapping</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="DisplayedScalarRangeLabel">
     <property name="text">
      <string>Displayed Range:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="qMRMLRangeWidget" name="DisplayedScalarRangeWidget">
     <property name="toolTip">
      <string>Set the display scalar range, used when the Display option is selected.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="ThresholdLabel">
     <property name="text">
      <string>Threshold:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="ThresholdCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Apply thresholding to your displayed model based on its scalar values. Some cells normals might be flipped, in which case you
should set &quot;backface&quot; and &quot;frontface&quot; to OFF in the Representation options to visualize all cells.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="ctkRangeWidget" name="ThresholdRangeWidget">
     <property name="toolTip">
      <string>Range used to threshold scalar values.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLRangeWidget</class>
   <extends>ctkRangeWidget</extends>
   <header>qMRMLRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkVTKDataSetArrayComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkVTKDataSetArrayComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
